lib/collection/src/config.rs

```diff
@@ -395,4 +396,28 @@ impl CollectionParams {
             Ok(Default::default())
         }
     }
+
+    pub fn scoring_order(&self, scoring_query: Option<&ScoringQuery>) -> CollectionResult<Order> {
```

Comment:
not sure if this function should be part of the collection configuration. I would rather prefer it independent